tests/test_audio.py

```diff
@@ -439,6 +439,12 @@ def test_audio_Audio_section(artist, album, track):
     assert track.section().key == album.section().key == artist.section().key
 
 
+def test_audio_Audio_sonicallySimilar(artist):
```

Comment:
```python
    similar_audio = artist.sonicallySimilar(limit=1)
    assert len(similar_audio) < 2

    similar_audio = artist.sonicallySimilar(maxDistance=0.1)
    assert all(i.distance <= 0.1 for i in similar_audio)
```
I have purposefully not added these tests as I think they are not in the scope of this api

Reply:
Add them so that we at least have coverage for the optional parameters. The results of the test will be meaningless.